tests/test_discord_bot.py

```diff
@@ -0,0 +1,104 @@
+from unittest.mock import AsyncMock
+from unittest.mock import Mock, patch
+from unittest import TestCase
+import pytest
+import json
+
+from gerousiabot import utils
+
+from gerousiabot import discord_bot
+
+logger = utils.setup_logger()
+
+
+bot_token = utils.get_env_variable('DISCORD_API_KEY')
+g_bot = discord_bot.GerousiaBot(bot_token=bot_token)
+
+class mock_status:
```

Comment:
[Nitpick] since these are classes, it should be `MockStatus`
https://www.python.org/dev/peps/pep-0008/#class-names